src/ducks/balance/Balance.jsx

```diff
@@ -432,6 +432,8 @@ class Balance extends PureComponent {
   }
 }
 
+const Balance = React.memo(_Balance)
```

Comment:
Is there really a gain to memo a PureComponent ?

Reply:
Je vais sortir des benchmarks, je pensais que ca ne pouvait pas faire de mal. Mais en fait React.Memo est fait pour les composants à base de fonctions... Je vais revoir ma copie pour voir si ca a vraiment du sens.

Reply:
Finalement les React.memo ne servaient à rien, merci de m'avoir challengé.